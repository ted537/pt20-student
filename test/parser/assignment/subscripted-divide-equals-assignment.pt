using input, output;
var a[1]: like 1;
a[1] = 15;
a[1] /= 3;
