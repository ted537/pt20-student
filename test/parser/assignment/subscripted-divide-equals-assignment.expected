 .sProgram
   .sIdentifier
   .sIdentifier
  % .sNewLine
  .sParmEnd
  .sBegin
    .sVar
     .sIdentifier
     .sArray
      .sInteger
       .sLike
       .sInteger
    % .sNewLine
    .sAssignmentStmt
    .sIdentifier
    .sSubscript
        .sInteger
    .sExpnEnd
     .sIdentifier
         .sInteger
     .sExpnEnd
     % .sNewLine
    .sAssignmentStmt
    .sIdentifier
    .sSubscript
        .sInteger
    .sExpnEnd
     .sIdentifier
         .sInteger
     .sDivide
     .sExpnEnd
     % .sNewLine
  .sEnd
